pkg/apis/eventing/v1alpha1/broker_types.go

```diff
@@ -70,6 +70,8 @@ type BrokerSpec struct {
 	// Broker. If left unspecified, it is set to the default Channel CRD for the namespace (or cluster, in case there
 	// are no defaults for the namespace).
 	// +optional
+	// Deprecated: See spec.config to configure aditional broker options.
+	// Unless class is <TBD> for channel based broker implementation.
```

Comment:
this seems worthy of a release note?

Reply:
Since we're deprecating this, should also be worthy of adding something to the status warning like for other deprecated fields in the past?

Reply:
It is not deprecated in the way that we have deprecated things in the past, and it will not be removed from v1alpha1. I think this is a housekeeping task for us to understand what gets trimmed out of the reconcilers when they move to v1beta1